src/homeWork_001/Diamond.java

```diff
@@ -0,0 +1,27 @@
+package homeWork_001;
+
+public class Diamond {
```

Comment:
create separate method, for example drawDiamond(int sizeOfDiamond). And if you write sizeOfDiamond = 3, you must get this picture: first line - one asterisk; second line - three asterisks, third line - one asterisks.